connector/saml/saml.go

```diff
@@ -72,6 +75,18 @@ func init() {
 	}
 }
 
+// normalizeBinding verifies binding from config and normalize the string
+func normalizeBinding(binding string) (string, error) {
+	binding = strings.ToLower(strings.TrimSpace(binding))
```

Comment:
I'd avoid any normalization. Keeps the configuration explicit.

Reply:
OK. I will remove this ToLower line.